src/sonic-eventd/debian/sonic-eventd.install

```diff
@@ -0,0 +1,2 @@
+usr/sbin/eventd
+usr/sbin/events_tool
```

Comment:
>usr/sbin [](http://example.com/codeflow?start=0&length=8)

We normally install executable to `/usr/bin`, you may check other debian packages we build. Could you follow?

Reply:
Done